tests/test_leaderboard.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+import pandas as pd
+import pytest
+
+from mteb.benchmarks import MTEB_EN, MTEB_ENG_CLASSIC, Benchmark, MTEB_multilingual
+from mteb.leaderboard.table import scores_to_tables
+from mteb.leaderboard.utils import filter_models, load_results
+
+# A couple of models to look for.
+SHOULD_BE_THERE = [
+    "NV-Embed-v2",
+    "e5-mistral-7b-instruct",
+    "text-embedding-005",
+    "voyage-large-2-instruct",
+    "cde-small-v2",
+    "SONAR",
+    "text-embedding-3-small",
+    "GritLM-7B",
+    "sentence-t5-xxl",
+    "M2V_base_glove",
+]
+BENCHMARKS_TO_CHECK = [
+    MTEB_EN,
+    MTEB_ENG_CLASSIC,
+    MTEB_multilingual,
+]
+
+all_results = load_results()
+
+
+@pytest.mark.parametrize("benchmark", BENCHMARKS_TO_CHECK)
+def test_leaderboard_tables(benchmark: Benchmark):
+    benchmark_results = benchmark.load_results(
```

Comment:
If we just want to ensure a tests for functionality we could create  `tests/dummy_results_repo/{model_name}/*.json`.

This would probably speed this up quite a bit.